lab-Marshall/.gitignore

```diff
@@ -0,0 +1,65 @@
+# Created by https://www.gitignore.io/api/osx,linux,node,vim
```

Comment:
For future assignments, you don't need to include the files from the project root folder in your folder (except maybe the eslintrc) - it duplicates the files unnecessarily. You can include the eslintrc file, but it'll probably be easier to just copy it to your home directory so that you don't need to copy it into your folder for each assignment.